harness-delegate.yaml

```diff
@@ -79,7 +79,7 @@ spec:
             scheme: HTTP
           initialDelaySeconds: 10
           periodSeconds: 10
-          failureThreshold: 20
+          failureThreshold: 3
```

Comment:
is this value 3 in manager code as well?

Reply:
in manager code is currently 2, i m changing that to 3 as well.
In helm it's 3